packages/javascript/bh-shared-ui/src/views/Explore/ContextMenu/AssetGroupMenuItemPrivilegeZonesEnabled.test.tsx

```diff
@@ -14,16 +14,73 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-import { Dialog } from '@bloodhoundenterprise/doodleui';
 import userEvent from '@testing-library/user-event';
 import { rest } from 'msw';
 import { setupServer } from 'msw/node';
-import { useState } from 'react';
-import { act, render, screen } from '../../../test-utils';
-import AssetGroupMenuItem from './AssetGroupMenuItemPrivilegeZonesEnabled';
+import { createAuthStateWithPermissions } from '../../../mocks';
+import { render, screen } from '../../../test-utils';
+import { Permission, apiClient } from '../../../utils';
+import { AssetGroupMenuItem } from './AssetGroupMenuItemPrivilegeZonesEnabled';
+
+const assetGroupTagsResponse = {
```

Comment:
The ContextMenu component was feeding many props to this component. Many of those values were only be used in this component, so they were relocated to be sourced within the component instead of from props. This required updating the mocks for the unit tests.